### ISSUE TITLE
Was working... now not working!

### ISSUE THREAD

**Issue by LordZork** [dgreif/ring#355]
No idea what happened... had updated to the latest v9.1.1, and it was working.  However, at some point it stopped.  I have tried everything, including re-flashing the MircoSD card with the latest Homebridge and started over, even removed My Home from the Home app and started over there too, and restarted everything, and instilled the ring plugin... same results...

```
[Ring] Request to https://app.ring.com/rhq/v1/devices/v1/locations failed with status 406. Response body: <html>
<head><title>406 Not Acceptable</title></head>
<body bgcolor="white">
<center><h1>406 Not Acceptable</h1></center>
<hr><center>openresty</center>
</body>
</html>

[Ring] Error connecting to API
[Ring] HTTPError: Response code 406 (Not Acceptable)
    at onResponse (/usr/local/lib/node_modules/homebridge-ring/node_modules/got/dist/source/as-promise/index.js:124:28)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  code: undefined,
  timings: {
    start: 1591760016362,
    socket: 1591760016365,
    lookup: 1591760016399,
    connect: 1591760016399,
    secureConnect: 1591760016474,
    upload: 1591760016475,
    response: 1591760016514,
    end: 1591760016517,
    error: undefined,
    abort: undefined,
    phases: {
      wait: 3,
      dns: 34,
      tcp: 0,
      tls: 75,
      request: 1,
      firstByte: 39,
      download: 3,
      total: 155
    }
  }
}

```

Even tried previous versions of the plugin... doesn't matter...

This is exhausting...

**Comment by dgreif** [comment]
Did you try generating a new refresh token?

**Comment by LordZork** [comment]
Yes, but that is also something you have to do with a fresh setup, so I had to do that again anyway after re-flashing and installing the plugin, etc.  Basically started from stratch (sadly, and probably unnecessarily).

That said, I just noticed this issue #337 and am wondering if that is what I am experiencing...

I am a total noob, but I was trying to get another plugin (a [wink](https://www.npmjs.com/package/homebridge-wink3) plugin) to work, and I think there was an issue that was sending Homebridge through a loop or something, so maybe Ring was getting hit too much, too fast, if that makes sense?  Based on what a couple guys were saying regarding #337, I am hoping I just need to wait 24 hours and it will work again.  Thoughts?

**Comment by dgreif** [comment]
That makes a lot of sense. Ring does have rate limiting in their api, and you probably hit it if your homebridge server was constantly crashing and restarting. For future reference, you can get a new refresh token by simply clearing out that field in config-UI-x. It will then switch immediately to the login form again. I’m sorry that isn’t documented anywhere, I’ll get it added to that screen. I would just turn off your homebridge server, take a break, and try again in 24 hours. Sorry for the headache!

**Comment by LordZork** [comment]
Thanks, and no worries.  I'm new and learning!  I will do as you suggested.

Also, I uninstalled that other plugin, and will wait until I hear back regarding the issue I submitted.  If I give that other plugin a try again, I will be sure to clear out the ring field in the config-UI-x and restart Homebridge first, so ring doesn't get hit like that again if I run into the same issue as before, with that other plugin.  I'm guessing that should keep me safe?
Thanks for the quick response!
